Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -399,13 +399,13 @@ bool RaycastForward( out RaycastHit closestHit )
 		Vector3 camPos = _cameraTransform.position;
 		Vector3 camForward = _cameraTransform.forward;
 
+	
 		Debug.DrawRay( camPos, camForward * _interactCheckDistance, Color.yellow, 1.0f, false );
 
-		RaycastHit[] hits = Physics.SphereCastAll(
-			new Ray( transform.position, camForward ),
-			_interactCheckRadius,
-			_interactCheckDistance,
-			_actor.cutting.cuttableLayer | _actor.inventory.buddyLayer );
+		RaycastHit[] hits = Physics.SphereCastAll( new Ray( camPos, camForward ),
+			                                       _interactCheckRadius,
+			                                       _interactCheckDistance,
+			                                       _actor.cutting.cuttableLayer | _actor.inventory.buddyLayer );

```

Comment:
This shouldn't have been reformatted, it was more readable before. Also, we should change this spherecast to happen from the player's position (rather than the camera's position) to deal with #58.